modules/openapi-generator/src/main/resources/JavaVertXWebServer/pojo.mustache

```diff
@@ -1,4 +1,4 @@
-@JsonInclude(JsonInclude.Include.NON_NULL) 
+@JsonInclude(JsonInclude.Include.NON_NULL){{>additionalModelTypeAnnotations}}
```

Comment:
Maybe the additional annotations should go above the hard-coded annotation to avoid issues if people override the template and don't begin with a newline?

Reply:
done